scale/recipe/models.py

```diff
@@ -1442,6 +1451,37 @@ def validate_recipe_type_v6(self, name, definition_dict):
 
         return RecipeTypeValidation(is_valid, errors, warnings, diff)
 
+    def validate_recursive_subs(self, recipe_type_name, definition, source_recipe_type_name=None):
+        """Validates the sub-recipes to ensure we don'' end up in a recipeception loop
+        :param recipe_type_name: The name of the recipe type we're looking for
+        :type recipe_type_name: String
+        :param definition: The RecipeDefinition we're searching
+        :type definition: RecipeDefinition
```

Comment:
Should do a full qualified python import so you can click through to definition in IDEs. Like so: https://github.com/ngageoint/scale/blob/master/scale/job/seed/manifest.py#L91

Reply:
fixed